lib/everypolitician/popolo.rb

```diff
@@ -55,16 +55,17 @@ def memberships
         @memberships ||= Memberships.new(popolo[:memberships], self)
       end
 
+      def elections
+        @elections ||= events.elections
+      end
+
       def legislative_periods
-        @legislative_periods ||= LegislativePeriods.new(
-          popolo[:events].select { |e| e[:classification] == 'legislative period' },
-          self
-        ).sort_by(&:start_date)
+        @legislative_periods ||= events.legislative_periods
       end
       alias terms legislative_periods
 
       def current_legislative_period
-        legislative_periods.last
+        legislative_periods.first
```

Comment:
I'm assuming this means that `legislative_periods` has now reversed order? That's not _necessarily_ a bad thing, but it definitely needs to be explicit in the CHANGELOG.